packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -102,6 +118,126 @@ describe("Integration tests", () => {
         expect(1).toBe(1);
```

Comment:
Any reasons this line is here

Reply:
Because on #129 there was a rename from db test to integration test